sql/amo_prod/fenix_addons_by_client_v1/query.sql

```diff
@@ -0,0 +1,75 @@
+CREATE TEMP function get_fields(m ANY TYPE) AS (
```

Comment:
```suggestion
CREATE TEMP FUNCTION get_fields(m ANY TYPE) AS (
```

Surprising that the formatting check didn't complain.